### DISCUSSION TITLE
Which ASN IP Address Family is associated with in Hegemony Alarms?

### DISCUSSION THREAD

**Discussion by moawnallah** [InternetHealthReport/gsoc#22] [General]
@romain-fontugne In Hegemony Alarms, what does the `af` attribute signify? Does it indicate that both `originasn` and `asn` share the same IP Address Family, such as IPv6, or does it pertain to just one of them, like `originasn` or `asn`?

```json
{
    "timebin": "2023-09-15T23:45:00Z",
    "originasn": 136941,
    "asn": 10075,
    "deviation": 32.7508335826017,
    "af": 6,
    "asn_name": "FGL-AS-BD Fiber@Home Global Limited, BD",
    "originasn_name": "UMS-AS-AP Unicom Multi System, BD"
}
```

**Comment by romain-fontugne** [comment]
that means that both are using IPv6 and the anomaly is seen in IPv6

**Reply by moawnallah** [reply]
Understood Thanks 🙏
